odoo/addons/base/tests/test_orm.py

```diff
@@ -35,10 +35,6 @@ def test_access_deleted_records(self):
         # Deleting an already deleted record should be simply ignored
         self.assertTrue(p1.unlink(), "Re-deleting should be a no-op")
 
-        # Updating an already deleted record should raise, even as admin
-        with self.assertRaises(MissingError):
-            p1.write({'name': 'foo'})
```

Comment:
Can you put this back?

Reply:
As discussed, unfortunately I cannot put it back, because it doesn't work as expected.

The assertion is supposedly testing that `write` should raise, but it is done on partner, which is a special case where `read` is called during `write`, and it is the `read` that leads to the exception (before this PR) and no exception (after this PR, since no query is done).